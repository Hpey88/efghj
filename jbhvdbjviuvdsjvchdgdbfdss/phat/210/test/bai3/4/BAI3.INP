5
4 2 45 8 9