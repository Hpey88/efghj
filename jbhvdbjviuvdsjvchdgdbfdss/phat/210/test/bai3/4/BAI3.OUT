2 4 8 9 45